packages/unuse-angular/src/index.ts

```diff
@@ -194,4 +198,15 @@ overrideUnResolveFn(unResolve);
 overrideTryOnScopeDisposeFn(tryOnScopeDispose);
 overrideIsUnRefFn(isUnRef);
 
-export { isUnRef, toUnSignal, tryOnScopeDispose, unResolve };
+const UNDEFINED = undefined;
+
+export {
+  isUnRef,
+  UNDEFINED as overrideIsUnRefFn,
```

Comment:
I think we need some comments why they should be `undefined` here.

Reply:
Done